bot.go

```diff
@@ -507,3 +510,39 @@ func (b *Bot) Respond(query Query, results []Result) error {
 
 	return nil
 }
+
+// Handle registers a handler for a message which text matches the provided regular expression
+func (b *Bot) Handle(command string, handler Handler) {
+	reg := regexp.MustCompile(command)
+	b.handlers[reg] = handler
+}
+
+// Serve listens for messages and route them to the appropiate handler
+func (b *Bot) Serve() {
+	messages := make(chan Message)
+	b.Listen(messages, 1*time.Second)
```

Comment:
I'm a bit new to the Telegram bot API. Is there a reason this is hard coded to 1 second? Do they do some sort of rate limiting? It would be nice if this would be configurable.